pkg/util/flavor/flavor.go

```diff
@@ -0,0 +1,24 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Could we add a method to set the flavor ? This would encapsulate the logic to the package instead of having external package set it manually. It doesn't change anything but would allow us to extend change the logic easily in the future without having to edit all the place were we do `flavor.AgentFlavor = "something"`.